store/index.js

```diff
@@ -5,13 +5,16 @@ const initialState = {
   language: 'en'
 };
 
-// Redux Reducer, this receives actions that are being DISPATCHED
+// Redux Reducer1, this receives actions that are being DISPATCHED
+// remove comment es-lint comment after setting action
+// eslint-disable-next-line no-unused-vars
 const reducer = (state = initialState, action) => {
-  if (action.type === 'CHANGE_LANGUAGE') {
-    return {
-      language: action.payload.language
-    };
-  }
+  // set action type here example:
+  // if (action.type === 'CHANGE_LANGUAGE') {
+  //   return {
+  //     language: action.payload.language
+  //   };
```

Comment:
should we remove the comments or do you think it's good to keep them?

Reply:
I removed it. Whoever will use redux will understand what are actions and TYPES.